client/src/main/java/org/spine3/protobuf/KnownTypes.java

```diff
@@ -204,9 +204,6 @@ private Builder putProperties(Properties properties) {
             return this;
         }
 
-        //TODO:2016-07-07:alexander.yevsyukov: Can we do what the below method does automatically?
-        // We can compose the same map on compile time by analyzing proto files packed with Protobuf's JAR.
-
         /**
```

Comment:
I remember that we found a way to get all Java classes from `protobuf` package using open-source Reflections lib. But then we removed this lib because we didn't want to depend on it because of one needed method. And we decided to just put needed proto classes (such as wrappers) to the map manually. 
Primarily, this was needed to support wrappers usage in tests. And that's why we put only wrappers to the map. Anyway, we should declare which classes to put (because we don't need all of them from that package), and the easiest way to do it is in `KnownTypes.java`.